src/ui/components/shared/SettingsModal/SettingsModal.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+const Modal = require("ui/components/shared/Modal").default;
+import SettingsNavigation from "./SettingsNavigation";
+import SettingsBody from "./SettingsBody";
+
+import { Settings, SelectedTab } from "./types";
+
+import "./SettingsModal.css";
+
+const settings: Settings = [
+  {
+    title: "Appearance",
+    items: [
+      {
+        label: "Dark Mode",
+        key: "appearance-dark-mode",
```

Comment:
If we use `.` as a delimiter that might help with nesting `appearance.dark-mode`